src/Services/utils.php

```diff
@@ -0,0 +1,92 @@
+<?php
```

Comment:
Assuming that these messages are not in the $error variable as mentioned above, perhaps we can add all of this as its own file in df-core Enums (like the verbmasks are in this file). That would make it more usable if other services need to use this list in future.